download-model.py

```diff
@@ -18,12 +18,16 @@
 
 import requests
 import tqdm
+from requests.adapters import HTTPAdapter
 from tqdm.contrib.concurrent import thread_map
 
 
 class ModelDownloader:
-    def __init__(self):
+    def __init__(self, max_retries):
```

Comment:
why doesn't it have a default value?! `def __init__(self, max_retries = 5):`

Reply:
I think this is not a library, but just a script, so I think there is no compatibility issue. Is this broke some old codes?